core/templates/services/exploration-stats-backend-api.service.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview TODO
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+import {
+  ExplorationStats,
+  ExplorationStatsObjectFactory,
+  IExplorationStatsBackendDict
+} from 'domain/statistics/ExplorationStatsObjectFactory';
+import { UrlInterpolationService } from
+  'domain/utilities/url-interpolation.service';
+
+@Injectable({providedIn: 'root'})
+export class ExplorationStatsBackendApiService {
+  constructor(
+      private explorationStatsObjectFactory: ExplorationStatsObjectFactory,
```

Comment:
Should indent by 2 not 4?

Reply:
Nope, parameters of functions are indented one more level than the function body itself.

Reply:
Hm in this case the function body doesn't really exist. (The additional indentation was to prevent confusion between the function args and function body, which isn't needed here.)

But, I'm happy either way. Thanks for clarifying!